app/presenters/transfer_package_presenter.rb

```diff
@@ -0,0 +1,47 @@
+class TransferPackagePresenter
+  attr_reader :transfer_package, :context, :current_user
+
+  def initialize(transfer_package, context)
+    @transfer_package = transfer_package
+    @context = context
+  end
+
+  # TODO(Rails 5.1): Use delegate_missing
+  delegate :uuid, :recipient, :confirmed_at, :confirmed?, :created_at, :receiver_institution, :sender_institution, :sample_transfers, :samples, to: :transfer_package
```

Comment:
You can use it today, I backported it in `lib/delegate_missing_to`!
```suggestion
  delegate_missing_to :transfer_package
```